lms/static/scripts/frontend_apps/components/ContentSelector.js

```diff
@@ -149,6 +149,7 @@ export default function ContentSelector({
           // An alias we maintain that provides multiple external documentation links for
           // different versions of Blackboard (Classic vs. Ultra)
           missingFilesHelpLink={'https://web.hypothes.is/help/bb-files'}
+          withBreadcrumbs
```

Comment:
Initially, I thought that maybe just having the breadcrumbs always render would be OK. However, the Dialog dimensions/styling within Canvas are such that there is fragility, and adding anything to the file dialog other than the file table messes up alignment. Looking forward to getting to the bottom of that.

Reply:
Canvas doesn't currently support folders, so it makes sense not to show breadcrumbs while that is the case.